src/playbook.yaml

```diff
@@ -9,33 +9,23 @@
       yum: name=* state=present update_cache=yes
 
     - name: install packages
-      package:
+      yum:
         name:
-          - "ntpdate"
           - "curl"
           - "python3-pip"
         state: present
 
-    - name: Prep snap classic
-      shell: "[ -x /snap ] || yum install -y epel-release snapd && systemctl enable --now snapd.socket && ln -s /var/lib/snapd/snap /snap || echo ''"
-      register: snap_installed
-
-    - name: Install awscli and amazon-ssm-agent
-      snap:
-        name:
-          - aws-cli
-          - amazon-ssm-agent
-        classic: yes
-      when: "not snap_installed.stdout"
-
-    - name: Update time
-      shell: "ntpdate -u pool.ntp.org"
-
-    - name: Make python3 the default python
-      shell: "update-alternatives --install /usr/bin/python python /usr/bin/python3 1"
+    # - name: Make python3 the default python
+    #   alternatives:
+    #     name: python
+    #     link: /usr/bin/python
+    #     path: /usr/bin/python3
```

Comment:
Are you removing this because python3 is already the default python alternative, or other?

Reply:
@tthyer I am removing this because for whatever reason when I update the python default, it doesn't allow me to ssh into the instance after creating the ec2 with the built AMI.